### ISSUE TITLE
[Development] - Command improvements for future usage

### ISSUE THREAD

**Issue by Unknown66** [eugenio412/PogomBOT#29]
## Command changes:
- `/set lang en`
- `/toggle show_text`
- `/toggle show_map`

This way we could easily add new configuration parameters and wouldn't need a new command every time.
`/set` and `/toggle` would print all possible options

**Comment by eugenio412** [comment]
yes, that could work, but I was thinking to a massive use of the inline keyboard for future iteration, I've used them a lot in my previous bots. That's also because the commands like "/command arg" is not that user friendly and it's not raccomanded in the bot API by telegram

**Comment by Unknown66** [comment]
Cool I was thinking about this too. :+1:
Then we would have only `/toggle` and `/set`. The user will be prompted after what todo with an inline keyboard right?

**Comment by eugenio412** [comment]
yes, that would be an idea, but we need to create a callback function for the inline keyboard callback, I's starting a branch so we can use it for this new feature without messing with the master.

Sure as hell I don't really understand how to use milesones

**Comment by kelt8** [comment]
how abt a command for current area coverage in a screenshot, that'll be nice
